lib/task-data/tasks/install-windows-2012.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Install Windows',
+    injectableName: 'Task.Os.Install.Win',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+            osType: 'windows', //readonly option, should avoid change it
+
+            profile: 'winpe.ipxe',
```

Comment:
Is 'winpe' naming still appropriate here? maybe 'window' or 'windowsserver' or some other nams more accurate?